### ISSUE TITLE
nadirpinhole not properly set in camera_solve with camera positions.

### ISSUE THREAD

**Issue by ScottMcMichael** [NeoGeographyToolkit/StereoPipeline#248]
When --camera-positions is passed in to camera_solve it should automatically use the session "nadirpinhole" but this is not working properly.

**Comment by oleg-alexandrov** [comment]
Good point. This would assume we are on Earth though, which I guess it is reasonable since we don't play with frame cameras on other planets.

There is also the issue if bundle_adjust assumes a datum when it does the first adjustment, while the cameras are still in an arbitrary coordinate system. Maybe camera_solve should invoke bundle_adjust with the --no-datum flag.

**Comment by ScottMcMichael** [comment]
I was not able to reproduce this error so it may take extra investigation.

**Comment by oleg-alexandrov** [comment]
This works as intended now. When `--camera-positions` is passed to `camera_solve` (via `--bundle-adjust-params`), the tool detects it and sets the session to `nadirpinhole` automatically, the same way it does when GCP files are supplied. It then calls `bundle_adjust` with `-t nadirpinhole --inline-adjustments`, and the camera positions are used to fit a similarity transform that moves the cameras from Theia's arbitrary frame into global ECEF coordinates.

I tested the exact `bundle_adjust` command `camera_solve` constructs for this case. The `nadirpinhole` session is accepted and the camera positions are read and applied without any complaint, so the original symptom no longer reproduces.
